script/main.py

```diff
@@ -243,17 +244,37 @@ def average_age(conn, gender):
     print('Average age of women: {value:.2f} years'.format(value = average_age_of_women))
   else:
     print('Overall average age: {value:.2f} years'.format(value = average_age_overall))
+
+
+def select_all_sorted_cities(conn):
+  cur = conn.cursor()
+  cur.execute("SELECT location_city, count(location_city) FROM users GROUP BY location_city ORDER BY count(location_city) DESC")
+  cities = cur.fetchall() 
+  return cities
+
+
+def most_popular_cities(conn, number):
+  cities = select_all_sorted_cities(conn)
+  i = 0
+  while i < int(number):
+    city_tuple = cities[i]
+    city_name = city_tuple[0]
+    city_occurrences = city_tuple[1] 
+    i += 1  
+    print('City {} occurr {} times'.format(city_name ,city_occurrences))
```

Comment:
you can use string interpolation here:
https://realpython.com/python-string-formatting/#3-string-interpolation-f-strings-python-36

Example
```python
print(f'City {city_name} occurr {city_occurrences} times')
```